ext/zend_test/tests/observer_bug81430_1.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Bug #81430 (Attribute instantiation frame accessing invalid frame pointer)
 --EXTENSIONS--
-zend_test
+zend-test
```

Comment:
Are these tests not run in CI or something? Wouldn't this have already been a failure?